scripts/spark-shell.sh

```diff
@@ -5,6 +5,6 @@ cd /usr/local/spark
 	--master spark://${SPARK_MASTER_PORT_7077_TCP_ADDR}:${SPARK_MASTER_ENV_SPARK_MASTER_PORT}  \
 	--conf spark.driver.host=${SPARK_LOCAL_IP} \
 	--properties-file /spark-defaults.conf \
-	--jars /spark-cassandra-connector_2.10-1.6.0.jar \
+	--jars /spark-cassandra-connector.jar \
```

Comment:
myślę że ten plik powinien mieć wersję w nazwie żeby jednym spojrzeniem wiedzieć co to jest za wersja

Reply:
racja. poprawiłem